src/main/kotlin/no/nb/bikube/catalogue/alma/controller/AlmaController.kt

```diff
@@ -84,12 +84,50 @@ class AlmaController(
             .map { marcXChangeService.writeAsByteArray(it) }
     }
 
+    @GetMapping("/isbn/{isbn}", produces = [MediaType.APPLICATION_XML_VALUE])
+    @Operation(summary = "Get bibliographic records by ISBN.")
+    @ApiResponses(value = [
+        ApiResponse(responseCode = "200", description = "OK"),
+        ApiResponse(responseCode = "400", description = "Invalid ISBN"),
+        ApiResponse(responseCode = "404", description = "ISBN not found"),
+        ApiResponse(responseCode = "500", description = "Server error")
```

Comment:
```suggestion
        ApiResponse(responseCode = "400", description = "Invalid ISBN", content = [Content()]),
        ApiResponse(responseCode = "404", description = "ISBN not found", content = [Content()]),
        ApiResponse(responseCode = "500", description = "Server error", content = [Content()])
```
Et lite tips for at swagger skal se litt mer rydding ut mtp. modellene. Uten dette forventer swagger samme modell på feilmeldinger som på 200. Får dessverre ikke lagt inn suggestion på import..